src/bp/nlu-core/intents/base-intent-classifier.ts

```diff
@@ -63,7 +39,12 @@ export class BaseIntentClassifier implements IntentClassifier {
 
     const exact_match_index = BuildExactMatchIndex(intents)
 
-    if (points.length <= 0) {
+    const allClasses = _(points)
```

Comment:
you could use uniqBy, it does the exact same thing I'm pretty sure but is 1 line instead of 3

Reply:
:+1: